rules-reviewed/eap6/websphere/websphere-catchall.windup.xml

```diff
@@ -17,6 +17,34 @@
         <tag>reviewed-2015-06-17</tag>
     </metadata>
     <rules>
+        <rule id="websphere-catchall-db2-00000">
+            <when>
+                <javaclass references="{package}.{type}" />
+            </when>
+            <perform>
+                <iteration>
+                    <when>
+                        <not>
+                            <has-hint />
+                        </not>
+                    </when>
+                    <perform>
+                        <hint title="IBM DB2 driver type reference" effort="1" severity="mandatory">
+                            <message>
+                                This is a reference to the IBM DB2 driver type (`{package}.{type}`). It probably does not need to be migrated. However the IBM DB2 driver should be configured properly.
+                            </message>
+                            <link href="https://access.redhat.com/solutions/93693" title="Configure datasource settings in EAP 6" />
+                        </hint>
+                    </perform>
+                </iteration>
+            </perform>
+            <where param="package">
+                <matches pattern="com\.ibm\.db2\.jcc(\..*)?" />
```

Comment:
whenever we know the package for a rule is fixed, it is better to use it directly and not in parametr as thjat results in better optimization for while scanning in tree hierarchy of packages.
So this should be better as:
`<javaclass references="com.ibm.db2.jcc.{subpackage}{type}" />` or straightforward as `<javaclass references="com.ibm.db2.jcc.{subpackage-with-type}" />`